app/change_sets/hyrax/work_change_set.rb

```diff
@@ -123,8 +129,27 @@ def work_members_json
       end.to_json
     end
 
+    def permissions_changed?
+      changed.include?('edit_users') ||
+        changed.include?('edit_groups') ||
+        changed.include?('read_users') ||
+        changed.include?('read_groups')
+    end
+
+    def visibility_changed?
+      changed['visibility']
```

Comment:
Are we cool with truthiness here, or should we make it explicit with `changed.key?('visibility')`?